### PR TITLE
Add redirectUri parameter

### PR DESCRIPTION
**Pull request by vishalshrm539**

### CODE REVIEW

**Comment by JEAxero** [comment]
Since we're adding a new optional config setting for redirectUri, we need to make sure that @syedi2 is informed so the doc can be updated. This may require another update to online docs or the SDK Guides. @vishalshrm539 @vrseraphin

**Comment by vishalshrm539** [comment]
FYI: Vinod wants Lease Plan to test these changes once before we merge them to the Main. So, this PR will be on hold until then.

**Review by vrseraphin — APPROVED** [review]

**Pull request merged**
